custom_components/rfplayer/number.py

```diff
@@ -50,3 +50,4 @@ async def async_set_native_value(self, value) -> None:
         rfplayer = self.hass.data[DOMAIN][RFPLAYER_PROTOCOL]
         await rfplayer.send_command_ack(command=int(value), protocol=self._protocol)
         self._state = value
+        self.async_write_ha_state()
```

Comment:
don't have to be awaited ?
```suggestion
        await self.async_write_ha_state()
```

Reply:
@Aohzan custom_components/rfplayer/switch.py and all homeassistant components call this function without the await. Hard to tell why as I am pretty new to the await syntax.

Reply:
Maybe the fact that the function definition does not have the async token before it explains why no await is added to the existing calls? 
From HomeAssistant core:
`homeassistant/helpers/entity.py:    def async_write_ha_state(self) -> None:`